index.html

```diff
@@ -0,0 +1,441 @@
+                  </a>
+                </div>
+                <div class="show-more">
+                  <a href="#">Show more</a>
+                </div>
+              </div>
+
+              <!-- Suggestion of who a user can follow -->
+              <div class="right-sidebar-to-follow">
+                <div class="section-text">
+                  <p><b>Who to follow</b></p>
+                </div>
+
+                <div>
+                  <div class="follow-card">
+                    <div>
+                      <img src="./amazing.jpg" alt="" class="user-avatar" />
+                    </div>
+                    <div>
+                      <div><b>Rasheed Adesokan</b></div>
+                      <div>@amazing_ng</div>
+                    </div>
+                    <div>
+                      <button><b>Follow</b></button>
+                    </div>
+                  </div>
+                  <div class="follow-card">
+                    <div>
+                      <img src="./amazing.jpg" alt="" class="user-avatar" />
+                    </div>
+                    <div>
+                      <div><b>Rasheed Adesokan</b></div>
+                      <div>@amazing_ng</div>
+                    </div>
+                    <div>
+                      <button><b>Follow</b></button>
+                    </div>
+                  </div>
+                  <div class="follow-card">
+                    <div>
+                      <img src="./amazing.jpg" alt="" class="user-avatar" />
+                    </div>
+                    <div>
+                      <div><b>Rasheed Adesokan</b></div>
+                      <div>@amazing_ng</div>
+                    </div>
+                    <div>
+                      <button><b>Follow</b></button>
+                    </div>
+                  </div>
+                </div>
+                <div class="show-more">
+                  <a href="#">Show more</a>
+                </div>
+              </div>
+
+              <!-- Terms of service and privacy area -->
+              <div class="right-sidebar-terms">
+                <div>
+                  <a href="#">Terms of Service</a><a href="#">Privacy Policy</a
+                  ><a href="#">Cookie Policy</a>
+                </div>
+                <div>
+                  <a href="#">Accessibility</a><a href="#">Ads info</a
+                  ><a href="#">More...</a>
+                </div>
+                <div><p>&copy; Twitter, Inc.</p></div>
+              </div>
+
+              <!-- Search bar area - top right -->
+              <div class="right-sidebar-search">
+                <div class="search-rounded">
+                  <img src="./twitter-icon/search-icon.svg" alt="" />
+                  <p>Search Twitter</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </main>
+    </div>
+  </body>
+</html>
```

Comment:
Nice clean flow of html with indents, easy to read and see the parent/child relations. Could perhaps use some more semantic labels rather than all being divs. This can help with accessibility features as well as further clarifying the regions of your code.